cabal/ganeti.template.cabal

```diff
@@ -49,7 +49,7 @@ library
     , pretty                        >= 1.1.1.0
     , process                       >= 1.1.0.1
     , random                        >= 1.0.1.1
-    , template-haskell              >= 2.7.0.0
+    , template-haskell              >= 2.11.0.0
```

Comment:
I know, actually I'm mentioning it at the bottom of the commit message. I can squash both commits in one if needed, but I chose to leave the first commit as-is because it's a cherry-pick (unfortunately as I used `git rebase -i --onto` the commits didn't get a `cherry-picked from` trailer).

Reply:
Initially I didn't get that from the commit message, the updated one all good. And in general, yes, don't squash as that loses history/context. Thanks!